### PR TITLE
Proof read and corrections in documentation

### PR DESCRIPTION
**Pull request by darrellwarde**

### CODE REVIEW

**Comment by neo4j-team-graphql** [comment]
Looks like you've updated the documentation!

Check out your changes at https://neo4j-graphql-docs-1009.surge.sh

File: docs/modules/ROOT/pages/auth/setup.adoc [hunk 1]
```diff
@@ -133,9 +135,11 @@ For example, you might have a function `decodeJWT` which returns a decoded JWT:
 ----
 const decodedJWT = decodeJWT(encodedJWT)
 
-const server = new ApolloServer({
-    schema: neoSchema.schema,
-    context: { jwt: decodedJWT.payload },
+neoSchema.getSchema().then((schema) => {
```
Comment (tbwiss) [inline docs/modules/ROOT/pages/auth/setup.adoc:138]: It's not on this line.
There seems to be a now outdated `neoSchema.schema` in 
`docs/modules/ROOT/pages/ogm/private.adoc`
`docs/modules/ROOT/pages/type-definitions/database-mapping.adoc`
`docs/modules/ROOT/pages/type-definitions/custom-directives.adoc`
Not sure we have to update them all, just wanted to point it out.

File: docs/modules/ROOT/pages/troubleshooting/index.adoc [hunk 2]
```diff
@@ -71,12 +71,14 @@ const neoSchema = new Neo4jGraphQL({
...
-server.listen().then(({ url }) => {
-  console.log(`Server ready at ${url}`);
+neoSchema.getSchema().then((schema) => {
+    const server = new ApolloServer({
+        schema,
+        context: ({ req }) => ({ req }),
+    });
+
+    server.listen().then(({ url }) => {
+    console.log(`Server ready at ${url}`);
```
Comment (danstarns) [inline docs/modules/ROOT/pages/troubleshooting/index.adoc:81]: I think there is a missing tab or space here.

File: docs/modules/ROOT/pages/driver-configuration.adoc [hunk 3]
```diff
@@ -236,8 +244,10 @@ const driver = neo4j.driver(
 
 const neoSchema = new Neo4jGraphQL({ typeDefs, driver });
 
-const server = new ApolloServer({
-  schema,
-  context: (req) => ({ driverConfig: { bookmarks: [req.headers["x-neo4j-bookmark"]] } })
+neoSchema.getSchema().then((schema) => {
+    const server = new ApolloServer({
+        schema,
+        context: (req) => ({ driverConfig: { bookmarks: [req.headers["x-neo4j-bookmark"]] } }),
```
Comment (danstarns) [inline docs/modules/ROOT/pages/driver-configuration.adoc:250]:
```suggestion
        context: ({ req }) => ({ driverConfig: { bookmarks: [req.headers["x-neo4j-bookmark"]] } }),
```

File: docs/modules/ROOT/pages/auth/setup.adoc [hunk 4]
```diff
@@ -98,9 +98,11 @@ Then, using Apollo Server as an example, you must include the request in the Gra
 
 [source, javascript, indent=0]
 ----
-const server = new ApolloServer({
-    schema: neoSchema.schema,
-    context: ({ req }) => ({ req }),
+neoSchema.getSchema().then((schema) => {
```
Comment (angrykoala) [inline docs/modules/ROOT/pages/auth/setup.adoc:101]: :clap:

File: docs/modules/ROOT/pages/mutations/create.adoc [hunk 5]
```diff
@@ -97,35 +97,36 @@ mutation {
...
+
+Consider the following type definitions:
 
-Consider the following type definition:
 [source, graphql, indent=0]
 ----
 type Actor {
     name: String!
-    movies: [Movie] @relationship(type: "ACTED_IN", direction: OUT)
+    movies: [Movie!]! @relationship(type: "ACTED_IN", direction: OUT)
```
Comment (tbwiss) [inline docs/modules/ROOT/pages/mutations/create.adoc:109]: It's not here. Found one more of the Many-to-* relationships typeDef updates:
https://github.com/neo4j/graphql/blob/dev/docs/modules/ROOT/pages/type-definitions/cypher.adoc line 189

**Review by tbwiss — APPROVED** [review]
LGTM now!

**Review by angrykoala — APPROVED** [review]

**Pull request merged**